qdrant-landing/content/articles/guide-filtering.md

```diff
@@ -0,0 +1,671 @@
+---
+title: "A Complete Guide to Filtering"
+short_description: "Merging different search methods to improve the search quality was never easier"
+description: "Learn everything about filtering in Qdrant. Discover key tricks and best practices to boost search performance and resource usage."
+preview_dir: /articles_data/guide-filtering/preview
+weight: -200
+author: Sabrina Aquino, David Myriel
+author_link: 
+date: 2024-09-01T00:00:00.000Z
+---
+
+<img src="/articles_data/guide-filtering/vector-search-ecommerce.png" alt="vector-search-ecommerce" width="80%">
+
+Imagine you sell computer hardware. To help shoppers easily find products on your website, you need to have a **user-friendly search bar**. If you’re selling computers and have extensive data on laptops, desktops, and accessories, your search feature should guide customers to the exact device they want - or a **very similar** match needed.
+
+When storing data in Qdrant, each product is a point, consisting of an `id`, a `vector` and `payload`:
+
+```json
+{
+  "id": 1, 
+  "vector": [0.1, 0.2, 0.3, 0.4],
+  "payload": {
+    "price": 899.99,
+    "category": "laptop"
+  }
+}
+```
+The `id` is a unique identifier for the point in your collection. The `vector` is a mathematical measure of similarity to other points in the collection. 
+Finally, the `payload` holds metadata that directly describes the point.
+
+Though we may not be able to decipher the vector, we are able to derive additional information about the item from its metadata, In this specific case, **we are looking at a data point for a laptop that costs $899.99**. 
+
+## What is filtering?
+
+When searching for the perfect computer, your customers may end up with results that are mathematically similar to the search entry, but not exact. For example, if they are searching for **laptops under $1000**, a simple vector search without constraints might still show laptops over $1000. 
+
+This is why semantic search alone **may not be enough**. In order to get the exact result, you would need to enforce a payload filter on the `price`. Only then can you be sure that the search results abide by the chosen characteristic.
+
+> This is called **filtering** and it is the most important feature of vector databases. 
+
+Here is how a **filtered vector search** looks behind the scenes. We'll cover its mechanics in the following section.
+
+<img src="/articles_data/guide-filtering/vector-search-filtering.png" alt="vector-search-ecommerce" width="80%">
+
+The filtered result is a combination of the semantic search and the filtering conditions imposed upon the query.
+
+#### What happens if you don't use filtering?
```

Comment:
I guess this section should be 'What happens if you don't use payload indexes?', since not using filtering would just return unrelevant data.